build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
         maven  {
             url "http://repo1.maven.org/maven2"
         }
+        maven { url "https://jitpack.io" }
```

Comment:
Is this import the one you need for the view you use in the dialog_custom_layout ?

Reply:
Yes, see https://github.com/chrisbanes/PhotoView for more information